manifest.json.template

```diff
@@ -53,13 +53,13 @@
       "explode": "true"
     },
     "org.zowe.explorer-jes": {
```

Comment:
Please back out these changes before merging

Reply:
changed to jes , uss , mvs latest staging version